packages/graphics/src/Graphics.ts

```diff
@@ -50,19 +79,50 @@ const DEFAULT_SHADERS = {};
  */
 export class Graphics extends Container
 {
+    /**
+     * Temporary point to use for containsPoint
+     *
+     * @static
+     * @private
+     * @member {PIXI.Point}
+     */
+
+    static _TEMP_POINT = new Point();
+
+    public shader: Shader;
+    public pluginName: string;
+
+    protected geometry: GraphicsGeometry;
```

Comment:
I don't think the access modifier for `geometry` should be `protected`. I think it should be `public readonly`

Reply:
we can't assign in readonly field. Blocked by TS.
`this.geometry = geometry || new Graphics Geometry();`
Or protected or public or change constructor or getter.

Reply:
I thought you could assign readonly in the constructor. Hmm. If that doesn't work, then private with getter I guess.

Reply:
Sorry, it is valid, assigment in constructor is really works. Thanks for insisting!

Reply:
But now we can't release it =) because destrou method assign null to propery, but in readonly!
Cool!

Reply:
Ok.
Now `geometry` is getter, `_geometry` is private.
API and JSDoc wasn't change (can get, can't set), but we can assign to `_geometry` internaly.